tests/output/test_io_mesh.py

```diff
@@ -147,7 +147,6 @@ def test_io_mesh_radial_hedgehog_extrusion(radial_hedgehog_mesh, tmpdir):
     _test_io_mesh_extrusion(radial_hedgehog_mesh, tmpdir)
 
 
-@pytest.mark.parallel(nprocs=3)
```

Comment:
This looks wrong.

Reply:
@ksagiyam suggested over in https://github.com/firedrakeproject/firedrake/pull/3122 that this test was never meant to pass in parallel

Reply:
Ah OK!